src/components/structures/ContextMenu.tsx

```diff
@@ -26,6 +26,7 @@ import { Writeable } from "../../@types/common";
 import { replaceableComponent } from "../../utils/replaceableComponent";
```

Comment:
Testing strategy:

 - ✅ `UserMenu` ([make sure can still navigate with custom user status messages](https://github.com/vector-im/element-web/issues/19881))
 - ✅ `MessageActionBar`
 - ✅ `ReactionsRow`
 - ✅ `RoomTile` -> `renderGeneralMenu`
 - ✅ `RoomTile` -> `renderNotificationsMenu`
 - :x: `RoomSubList` (the settings `...` on each room list heading for sorting stuff)
    - Something is very strange with this menu. On `develop`, it requires 2 clicks to change the radios and you can't navigate to the checkboxes. Also when using arrow keys on the radios, it seems to the opposite of the focused selection.
    - In this PR, I can't even get the radios in the menu to work with my mouse. It seems to only re-render when I check/uncheck the checkboxes below.
 - ✅ `ThreadPanelHeader` "My threads" vs "All threads" (accessed via the top bar, not a thread in the timeline)
 - ✅ `RoomListHeader` (the `+` next to the "Rooms" heading)
 - ✅ `NetworkDropdown` in the explore rooms modal
 - ✅ Spaces `QuickSettings` (under feature flag, at bottom of Spaces panel)
 - ✅ `SpaceCreateMenu`
 - ❔ `WidgetCreateMenu`
    - Untested, what's a good way to test?
 - ❔ `CallContextMenu`
    - Untested, what's a good way to test?
 - ❔ `DialpadContextMenu`
    - Untested, what's a good way to test?

Reply:
What is `WidgetCreateMenu`?

`CallContextMenu` start a 1:1 call (has to be answered) to see this button labelled `More`
`DialpadContextMenu` I suggest editing the code to make https://github.com/matrix-org/matrix-react-sdk/blob/7b94e13a8438f98dfd6b7543d011fc8835968d38/src/components/views/voip/CallView/CallViewButtons.tsx#L216 render regardless of dtmf support

Reply:
What is `WidgetCreateMenu`?

Typo: `WidgetContextMenu`

Reply:
`WidgetContextMenu`:

1. Room info `i` top right
2. `Add widgets, bridges & bots`
3.  add something like an Etherpad
4.  close modal
5.  Hover over etherpad
6. Click ... menu

Reply:
* DialpadContextMenu - this looks broken on develop - it hangs off the right of the screen
 * CallContextMenu - to test, place a call & click the '...' button (should just get 'hold')
 * WidgetCreateMenu (WidgetContextMenu?) - I think this is just the top right '...' button on any widget

Reply:
Thanks for the tips @t3chguy 😀

Tested `CallContextMenu` ✅

Tested `DialpadContextMenu` but it looks like it's not keyboard accessible before ✖

Tested `WidgetContextMenu` ✅

> `WidgetContextMenu:`
> ...

Ahh I was thinking about how I can get it to work for a local homeserver, `Cannot connect to integration manager` -> `Cannot connect to integration manager`

But I can just sign in on `matrix.org` 🎉

Reply:
> DialpadContextMenu - this looks broken on develop (and app) - it hangs off the right of the screen

Ye I noticed this too! It's a bit hard to make it fully visible but I made it work.

Thanks for the tips @dbkr 😀

Reply:
Testing strategy:

 - ✅ `UserMenu` ([make sure can still navigate with custom user status messages](https://github.com/vector-im/element-web/issues/19881))
 - ✅ `MessageActionBar`
 - ✅ `ReactionsRow`
 - ✅ `RoomTile` -> `renderGeneralMenu`
 - ✅ `RoomTile` -> `renderNotificationsMenu`
 - :x: `RoomSubList` (the settings `...` on each room list heading for sorting stuff)
    - Something is very strange with this menu. On `develop`, it requires 2 clicks to change the radios and you can't navigate to the checkboxes. Also when using arrow keys on the radios, it seems to the opposite of the focused selection.
    - In this PR, I can't even get the radios in the menu to work with my mouse. It seems to only re-render when I check/uncheck the checkboxes below. Any tips?
 - ✅ `ThreadPanelHeader` "My threads" vs "All threads" (accessed via the top bar, not a thread in the timeline)
 - ✅ `RoomListHeader` (the `+` next to the "Rooms" heading)
 - ✅ `NetworkDropdown` in the explore rooms modal
 - ✅ Spaces `QuickSettings` (under feature flag, at bottom of Spaces panel)
 - ✅ `SpaceCreateMenu`
 - ✅ `WidgetContextMenu`
 - ✖ `CallContextMenu`
    - Was not keyboard accessible before (tested on `develop`) and is hangs off on the right-side of the screen
 - ✅ `DialpadContextMenu`

Reply:
> But I can just sign in on matrix.org

Could run `/addwidget https://something` and then follow steps 5-6 as before

Reply:
I can reproduce the RoomSubList weirdness, additionally I can only *rove* between the radios, going down continually just loops. It also seems like the radio interactions don't actually take effect when I am on the Home page. I managed to reproduce a similar issue on a local develop build once but for some reason on your Netlify its 90% of the time.

https://user-images.githubusercontent.com/2403652/146210678-6a523f8c-c313-434c-b6d3-1b60ff423432.mov

Reply:
Thanks for looking into it @t3chguy. It's nice to see another reproduction to know it's not just me.

I dove into this more and fixed it in https://github.com/matrix-org/matrix-react-sdk/pull/7353#discussion_r769656166